tests/mypy/configs/mypy-plugin-strict-no-any.ini

```diff
@@ -0,0 +1,23 @@
+[mypy]
```

Comment:
can't we just use `strict = true`?

Reply:
no unfortunately because `strict` doesn't enable any of the `Any` rules